src/Keboola/StorageApi/Client.php

```diff
@@ -543,6 +543,7 @@ public function createTable($bucketId, $name, CsvFile $csvFile, $options = array
             "primaryKey" => isset($options['primaryKey']) ? $options['primaryKey'] : null,
             "columns" => isset($options['columns']) ? $options['columns'] : null,
             "data" => fopen($csvFile->getPathname(), 'r'),
+            'syntheticPrimaryKeyEnabled' => isset($options['syntheticPrimaryKeyEnabled']) ? $options['syntheticPrimaryKeyEnabled'] : null,
```

Comment:
OptionsResolver by nebyl časem asi špatný

Reply:
Tak to rozhodně.